lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1050,6 +1239,37 @@ module Resources
               'display': 'Emergency Contact'
             }
           ]
+        },
```

Comment:
We might have to remove some of this when it comes time to release (this is the write example). There's a chance that not all of the implementation that is documented will be released to prod on the same day.